layers/meta-resin-up-board/recipes-core/images/resin-image-flasher.bbappend

```diff
@@ -0,0 +1,7 @@
+include resin-image.inc
+
+# Put the initramfs inside the boot partition
```

Comment:
so where do you put the initramfs inside the boot partition? I see you only copy gfub config files here